Sources/Starknet/Providers/StarknetProvider/JsonRpcParams.swift

```diff
@@ -63,4 +63,35 @@ struct GetClassHashAtParams: Encodable {
     }
 }
 
+public struct Filter: Encodable {
```

Comment:
Let's rename it to `StarknetGetEventsFilter`, to indicate in the name that it's used by this particular method

Reply:
And I think it could also be moved to `Events.swift` file